packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionChannelTypesMixin.ts

```diff
@@ -16,13 +16,13 @@ const allowedChannelTypes = [
 
 export type ApplicationCommandOptionAllowedChannelTypes = typeof allowedChannelTypes[number];
 
-const channelTypesPredicate = z.array(
-	z.union(
-		allowedChannelTypes.map((type) => z.literal(type)) as [
-			ZodLiteral<ChannelType>,
-			ZodLiteral<ChannelType>,
-			...ZodLiteral<ChannelType>[]
-		],
+const channelTypesPredicate = s.array(
+	s.union(
+		...(allowedChannelTypes.map((type) => s.literal(type)) as [
+			LiteralValidator<ChannelType>,
+			LiteralValidator<ChannelType>,
+			...LiteralValidator<ChannelType>[]
+		]),
 	),
```

Comment:
```suggestion
	s.union(...allowedChannelTypes.map((type) => s.literal(type))),
```
This cast is no longer needed.